### PR TITLE
navigation panel

### PR DESCRIPTION
**Pull request by ddfridley**

### CODE REVIEW

**Comment by ddfridley** [comment]
@tngan8519 It's really great that you went into so much detail creating stories for the different conditions, and putting so much detail into get####Status functions.  I want to invite @BolunThompson to look at the get###Status functions in [navigation-panel.js](https://github.com/EnCiv/undebate-ssp/blob/navigation-panel%2316/app/components/navigation-panel.js) like get getInvitationStatus - are these useful or is there commonality in getting the status for the Submissions component #56 ?  If so lets discuss.  

- [ ] please pull main and merge it in to this branch
- [ ] The branch is missing assets/svg/right-arrow can you add it?
- [ ] In this test, http://localhost:6006/?path=/story/navigation-panel--submission-when-reminder-is-sent, when I click on Election Table - it doesn't change in a way to indicate I've clicked on it.  On the other categories, they turn gray.  This one is already red.  I think it should either turn a darker red, or turn grey but have a red outline.  This probably generalizes to other categories. 

This is looking really good thanks!

**Comment by ddfridley** [comment]
- [ ] When mouse hovers over an election category, the cursor should change to a pointer.

File: stories/navigation-panel.stories.js [hunk 1]
```diff
@@ -0,0 +1,1320 @@
...
+
+export const ElectionComplete = Template.bind({})
+ElectionComplete.args = {
+    defaultElectionObj: {
+        _id: '2349099238402',
+        electionName: 'U.S Presidential Election',
+        organizationName: 'United States Federal Government',
+    },
+}
+
```
Comment (ddfridley) [inline stories/navigation-panel.stories.js:41]: I have a suggestion to shorten this file.  create 
```
const defaultElectionObj={ ...} in the file.
 ElectionComplete.args={ defaultElectionObj: {...defaultElectionObj, {timeline: {a different timeline}}}
```
this way you won't have to repeat stuff that doesn't change, and you focus attention on what's different.
If you wanted to go further down this path you could use
````
import { merge } from 'lodash'
ElectionComplete.args={
    defaultElectionObj: merge({},defaultElectionObj, 
          {timeline: { moderatorDeadlineReminderEmails:  { date: 'a different date' } }}
     )
}
```
This is not a required change.  But I wanted to mention it for future thinking.  
Thanks for your hard work!

**Comment by BolunThompson** [comment]
It looks like the get###Status functions have some commonalities with getting the status for Submissions component #56. Maybe they should be added to electionMethods or somewhere similar?

**Comment by tngan8519** [comment]
> * [ ]  When mouse hovers over an election category, the cursor should change to a pointer.

I add change to  a pointer. Do you need to change the background color of the election category too?

**Pull request merged**